src/index.json

```diff
@@ -3512,6 +3512,7 @@
                     },
                     "extras": [],
                     "generator": "bdist_wheel (0.30.0)",
+                    "keywords": ["az", "sql", "arcdata", "cli", "azure", "sqlmi", "dc"],                    
```

Comment:
Why need you update the keywords of application-insights?

Reply:
@kairu-ms - These come from setup.py/dist-info/METADATA  hence needed for `./scripts/ci/test_index.py` to pass

Is the convention/ask to leave `keyword` blank for exts?

Thanks in advance.

Reply:
You are changing the keywords of application-insights which is an extension owned by another team.

Reply:
I see... Let us remove the keywords to dedicated values and push again. Thanks.

Reply:
@kairu-ms - I removed the offending keywords as suggested. When time permits please review one more time. Thank you.

Reply:
@swells The keywords are in wrong place. Your extension is arcdata, but you are adding keywords in application-insights extension.
```suggestion
```